mvm/machine/signer.go

```diff
@@ -4,76 +4,86 @@ import (
 	"bytes"
 	"context"
 	"encoding/binary"
-	"encoding/hex"
+	"fmt"
+	"sort"
 	"time"
 
 	"github.com/MixinNetwork/mixin/logger"
 	"github.com/MixinNetwork/tip/crypto"
 	"github.com/MixinNetwork/tip/crypto/en256"
+	"github.com/MixinNetwork/trusted-group/mvm/constants"
 	"github.com/MixinNetwork/trusted-group/mvm/encoding"
 	"github.com/drand/kyber/sign/tbls"
 )
 
+func (m *Machine) getProcessInfo(processId string) (string, string, bool) {
```

Comment:
This could just use RLock(). And name it with getProcess(pid) with a single *Process return value could be easier.

Reply:
I Haven't seen we need a RLock currently, but yes let's use a RLock to avoid some potential deadlock.